rootfs/templates/wrapper

```diff
@@ -84,13 +84,29 @@ function use() {
     # Implement DNS fix related to https://github.com/docker/docker/issues/24344
     DOCKER_ARGS+=("--dns=${DOCKER_DNS}")
   fi
-
+     
   if [ -n "${HOME}" ]; then
     if [ "${HOME}" == "/localhost" ]; then
       echo "WARNING: not mounting ${HOME} because it conflicts with geodesic"
     else
-      echo "# Mounting ${HOME} into container"
-      DOCKER_ARGS+=(--volume=${HOME}:/localhost)
+      if  [[ $(cat /proc/version) =~ Microsoft ]] ; 
+      then
+        windowsUserName=$(/mnt/c/Windows/System32/cmd.exe /c 'echo %USERNAME%' | tr -d '\r')
+
+        if [ -d "/mnt/c/Users/${windowsUserName}/AppData/Local/Packages/CanonicalGroupLimited.Ubuntu18.04onWindows_79rhkp1fndgsc" ]; 
```

Comment:
> CanonicalGroupLimited.Ubuntu18.04onWindows_79rhkp1fndgsc
Experience tells me this is probably related to a build number or CPU architecture or both. This won't work. 

Instead, do something like this:
```
for dir in /mnt/c/Users/${windowsUserName}/AppData/Local/Packages/*Ubuntu*on*Windows*; do
  if [ -d "${dir}/LocalState/rootfs${HOME}" ]; then 
     ...
     break
  fi
done
```